src/app/pages/questions/services/finish-task.service.ts

```diff
@@ -84,4 +84,11 @@ export class FinishTaskService {
   getTimeCompleted(answers) {
     return answers[answers.length - 1].endTime
   }
+
+  cancelNotificationsForCompletedTask(task): Promise<any> {
+    const notifications = task.notifications
+    return notifications.forEach(n =>
+      this.config.cancelSingleNotification(n.id)
```

Comment:
why not cancel them in a single request? by sending list of notifications?

Reply:
This is so that we get the result if a single notification it's successfully deleted, then we can update the notification in the app accordingly.